packages/Liquid-Tests.package/LQPasswordGuardTest.class/instance/testPasswordGuard.st

```diff
@@ -0,0 +1,9 @@
+as yet unclassified
+testPasswordGuard
+	| guard |
+	guard := LQPasswordGuard newWithPassword: 'pikachu'.
```

Comment:
[pebble] The same comment as for `testTwoHundredUsers` applies here, however in this scenario it is not as bad to choose your own password. Still, it suggests to readers that this is something people can do, whilst the code breaks if done like this. Since there would be a lot of unnecessary overhead (starting a Poll) if done "correctly", I would suggest just adding a comment stating the fact that passwords should be randomly generated.